frontend/templates/troubleshooting/hooks/GuideModel.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Auto-generated by https://app.quicktype.io/ based on the JSON from
+ * https://agilbert.cominor.com/api/2.0/Guides/39258
+ */
+export type Guide = {
+   conclusion_raw: string;
+   conclusion_rendered: string;
+   difficulty: string;
+   documents: any[];
+   flags: Flag[];
+   guideid: number;
+   image: ImageElement;
+   introduction_raw: string;
+   introduction_rendered: string;
+   featured_document_embed_url: string;
+   featured_document_thumbnail_url: string;
+   locale: string;
+   parts: Part[];
+   prerequisites: any[];
+   steps: Step[];
+   subject: string;
+   summary: string;
+   time_required: string;
+   time_required_min: number;
+   time_required_max: number;
+   title: string;
+   tools: Part[];
+   type: string;
+   revisionid: number;
+   created_date: number;
+   published_date: number;
+   modified_date: number;
+   prereq_modified_date: number;
+   public: boolean;
+   comments: GuideComment[];
+   category: string;
+   url: string;
+   patrol_threshold: number;
+   can_edit: boolean;
+   favorited: boolean;
+   author: GuideAuthor;
+   langid: string;
+   featured_documentid: null;
+   intro_video_url: null;
+   intro_video: null;
+   completed: boolean;
+};
+
+export type GuideAuthor = {
+   userid: number;
+   username: string;
+   unique_username: string;
+   join_date: number;
+   image: PurpleImage;
+   reputation: number;
+   url: string;
+   teams: number[];
+};
+
+export type PurpleImage = {
```

Comment:
Purple?

Reply:
The type autogenerator chose weird adjectives for some of these; I can only assume there was some value that it tried to infer from.